charts/oes/charts/spinnaker/config/orca-local.yml

```diff
@@ -286,7 +300,9 @@ job:
                         value: "${{execution.startTime} .toString()}"
                       - name: user
                         value: "${execution.authentication.user}"
-                    image: 'opsmxdev/policy:v1'
+                      - name: executionid
+                        value: "${execution.id}"
+                    image: 'opsmxdev/policy:v3.7'
```

Comment:
this policy image should have its own versioning. It needn't align with our OES version. Please maintain the Dockerfile used to create this image in some known directory. May be, you can put under this enterprise-spinnaker repo under Dockerfiles